app/src/lib/git/commit.ts

```diff
@@ -34,3 +34,7 @@ export async function createCommit(repository: Repository, message: string, file
     }
   }
 }
+
+export async function revertCommit(repository: Repository, SHA: string) {
```

Comment:
We've generally organized functionality into files by the git command it calls, which means this should be in `revert.ts`. (Also could use a lil documentation.)